BP/history/provinces/bp_k_macedonia.txt

```diff
@@ -1,19 +1,35 @@
-﻿# k_macedonia
+﻿# Notes: 
+# Latest possible attestation of Illyrian is the 6th century, more definitely 4th-5th Century (St. Jerome)
+# https://en.wikipedia.org/wiki/Illyrian_language#Era
+# Albanian arises sometime around 6th-7th century (albanian)
+# https://en.wikipedia.org/wiki/Koman_culture (Another example for continuity)
+# 
+#
+# So splits from Illyrian in 550 (in modern albania; probably proto-Albanian but close enough)
+# 550.1.1: Rise of Albanian culture in modern Albania
+# 500.1.1: First wave of permanent Hellenization & Hello-Thracianization of Thracian provinces
+#
+# Redid Illyrian/Thracian division based on this map:
+# https://en.wikipedia.org/wiki/Thracians#/media/File:Odrysian.svg
```

Comment:
Isn't that one a bit old?

Reply:
The map is old, but it tells us the maximal extent of Thracian culture to better define the border between Illyrian and Thracian cultures.